### PR TITLE
Merge Q4 features and fixes into Master

### PR DESCRIPTION
**Pull request by danielfx90**
### Important Notes
- This is the version deployed during the first week of January. It is also tagged as `deploy-new-year`.
- Subsequent deploys during Q1 will only have fixes, but not new features, without counting the integration with travis #108 
- All fixes to be done and deployed during Q1 should branch out from `master` and go through a PR pointed to that same target.
- From the moment this PR is merged, `develop` will be the base branch for the implementation of Protocol V2.

### CODE REVIEW

File: buyer-signing-service/src/server.js [hunk 1]
```diff
@@ -2,11 +2,14 @@ import 'babel-polyfill';
...
 import logger from './utils/logger';
+import writePid from './utils/writePid';
 
 const server = () => {
   const { port, host, env } = config;
-  app.listen({ port, host }, () =>
-    logger.info(`Buyer Signing Service listening on port ${port} and host ${host} in ${env} mode`));
+  app.listen({ port, host }, () => {
+    logger.info(`Buyer Signing Service listening on port ${port} and host ${host} in ${env} mode`);
+    writePid(`${__dirname}/../service.pid`);
```
Comment (manelli) [inline buyer-signing-service/src/server.js]: Why is this necessary?

Comment (danielfx90) [reply buyer-signing-service/src/server.js]: I'm not sure why is there at all or when we did it. It is handy, though.
